solver/src/http_solver.rs

```diff
@@ -272,6 +289,22 @@ impl HttpSolver {
         serde_json::from_str(text.as_str())
             .with_context(|| format!("failed to decode response json, {}", context()))
     }
+
+    async fn order_cost(&self) -> Result<u128> {
+        // Q: should we use estimate_with_limits below instead?
+        // Q: are the units (gwei) correct?
+        let gas_price_gwei = self.gas_price_estimator.estimate().await? * 1e9;
```

Comment:
I think the price estimator returns the price in WEI so no need to multiply with 1e9